kiwi/defaults.py

```diff
@@ -811,6 +811,7 @@ def get_shim_loader(root_path: str) -> Optional[shim_loader_type]:
             shim_pattern_type('/usr/lib/shim/shim*.efi.signed', 'shimx64.efi'),
             shim_pattern_type('/usr/share/efi/*/shim.efi', None),
             shim_pattern_type('/usr/lib64/efi/shim.efi', None),
+            shim_pattern_type('/boot/efi/EFI/*/shimx64.efi', None),
```

Comment:
This will break for non-x64 arches for CentOS.

Reply:
You mean it will work only for x64 correctly?

What is about adding concrete filenames for all arches (`shimia32.efi`, `shimaa64.efi`, etc.) before the wildcard line matches?